extensions/expression-language/tests/ExpressionLanguageTests.cpp

```diff
@@ -1321,13 +1321,13 @@ TEST_CASE("Now Date", "[expressionNowDate]") {
 TEST_CASE("Parse RFC3339 with Expression Language toDate") {
   using date::sys_days;
   using org::apache::nifi::minifi::utils::timeutils::parseRfc3339;
-  using namespace date::literals;
+  using namespace date::literals;  // NOLINT(google-build-using-namespace)
   using namespace std::literals::chrono_literals;
   using std::chrono::milliseconds;
 
-  milliseconds expected_second = std::chrono::floor<milliseconds>((sys_days(2023_y / 03 / 01) + 19h + 04min + 55s).time_since_epoch());
-  milliseconds expected_tenth_second = std::chrono::floor<milliseconds>((sys_days(2023_y / 03 / 01) + 19h + 04min + 55s + 100ms).time_since_epoch());
-  milliseconds expected_milli_second = std::chrono::floor<milliseconds>((sys_days(2023_y / 03 / 01) + 19h + 04min + 55s + 190ms).time_since_epoch());
+  milliseconds expected_second = std::chrono::floor<milliseconds>((sys_days{2023_y / 03 / 01} + 19h + 04min + 55s).time_since_epoch());  // NOLINT(whitespace/braces)
+  milliseconds expected_tenth_second = std::chrono::floor<milliseconds>((sys_days{2023_y / 03 / 01} + 19h + 04min + 55s + 100ms).time_since_epoch());  // NOLINT(whitespace/braces)
+  milliseconds expected_milli_second = std::chrono::floor<milliseconds>((sys_days{2023_y / 03 / 01} + 19h + 04min + 55s + 190ms).time_since_epoch());  // NOLINT(whitespace/braces)
```

Comment:
What was the error here?

Reply:
I think the clang-tidy check returned false positive here as it recognized this call as a C-style cast and warned for it, this change was meant to avoid that warning.

Reply:
And what necessitated `// NOLINT(whitespace/braces)`?

Reply:
That's also a false positive as it wants to add a space after the brace in `sys_days{`

Reply:
Reverted in b1d4a35c9a2021f2bd8d0fa750e2f2971f635cae